packages/shared/formatProdErrorMessage.js

```diff
@@ -11,10 +11,14 @@
 // during build.
 
 function formatProdErrorMessage(code) {
-  let url = 'https://reactjs.org/docs/error-decoder.html?invariant=' + code;
-  for (let i = 1; i < arguments.length; i++) {
-    url += '&args[]=' + encodeURIComponent(arguments[i]);
+  let url = 'https://react.dev/errors/' + code;
+  if (arguments.length > 1) {
+    url += '?args[]=' + encodeURIComponent(arguments[1]);
```

Comment:
Why can't we append the arguments in the same for loop that is running below, we can start from `1` index as it was earlier.

Reply:
Yeah no idea why I did this, that's how it was done before, I'll revert